climateconnect/climateconnect
# Fix: Organization description fields not saving from check translations

## Description

Closes #1045

Modified `EditOrganizationRoot.js`.  An issue was that when trying to make changes from the check translations interface, the new values entered would never update upon clicking "save". 
The reason for this was that there was a check being made between a "new" organization and an "old" one, however the old one was also always being updated when inputting new text into the field, thus no changes were ever detected.

Solution: Make an API call to get the unchanged state of the organization and compare that to the new one. 

Implemented a fix for a visual bug while viewing translations for organizations, where the left textfield in the about section box size would be shorter than the one above it.
Updated button label for when editing project with automatic translations from "Publish" to "Save".
Updated buttons label and help text to match that of the selected locale.


## Test plan
- Sign into an account that has admin rights within an organization
- Go to that organization's page
- Go to edit organization
- Go to check translations
- Enter new text into the short description or about fields
- Press save
- New text should be displayed

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
